src/otx/core/config/data.py

```diff
@@ -90,6 +90,24 @@ class VisualPromptingConfig:
     use_point: bool = False
 
 
+@dataclass
+class UnlabeledDataConfig(SubsetConfig):
+    """DTO for unlabeled data."""
+
+    data_root: str | None = None
+    data_format: str = "image_dir"
+
+    batch_size: int = 0
+    subset_name: str = "unlabeled"
```

Comment:
Do we need subset name in config since it is always should be unlabeled?

Reply:
I'll remove this part. thank you!

Reply:
I've been working on this, but there's a problem. Currently, `UnlabeledDataConfig` is a structure that inherits from `SubsetConfig`, so it needs that attribute. I'll leave it as it is for now.